Cargo.toml

```diff
@@ -3,6 +3,7 @@ members = [
     "serde-name",
     "serde-reflection",
     "serde-generate",
+    "serde-generate/serde-generate-bin",
```

Comment:
nit: Can you move `serde-generate-bin` to the root of the repository? (for better discoverability)